src/main/resources/application.yml

```
@@ -0,0 +1,15 @@
+spring:
+  application:
+    name: Book-Library
+
+  # Database
+  datasource:
+    driver-class-name: org.postgresql.Driver
+    url: jdbc:postgresql://localhost:5432/librarydb
+    username: postgres
+    password: 123
```

```suggestion
    url: jdbc:postgresql://localhost:5432/book-library-yev
    username: postgres
    password: postgres
```
Пропиши пожалуйста такие проперти!
Это чисто для меня, для удобства локального запуска. У меня уже много БД, от каждого студента по одной. Все с одним паролем )